blocks/class-decent-comments-blocks.php

```diff
@@ -80,9 +80,9 @@ public static function decent_comments_block_init() {
 
 		wp_register_style(
 			'decent-comments-block-style',
-			plugins_url( 'decent-comments/src/style-index.css', __FILE__ ),
+			plugins_url( 'decent-comments/build/index.css', __FILE__ ),
 			array(),
-			filemtime( plugin_dir_path( __FILE__ ) . 'decent-comments/src/style-index.css' )
+			filemtime( plugin_dir_path( __FILE__ ) . 'decent-comments/build/index.css' )
```

Comment:
use plugin version constant `DECENT_COMMENTS_PLUGIN_VERSION` instead of `filemtime(...)`